space_dodge/main.py

```
@@ -47,7 +47,7 @@
 # Import all the files( images, sounds, etc. )
 from file_handling.constants_and_file_loading import (muteImage, unmuteImage, pauseButtonImage, game_background,
                                                       sadSound, GameOverSound, highscoreSound)
-from file_handling.loading_func import load_highscore
+from file_handling.loading import load_highscore
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)